integration/combination/test_function_with_sns.py

```diff
@@ -1,6 +1,10 @@
+from unittest.case import skipIf
+
 from integration.helpers.base_test import BaseTest
+from integration.helpers.resource import current_region_does_not_support
 
 
+@skipIf(current_region_does_not_support(["SQS"]), "SQS is not supported in this testing region")
```

Comment:
Test is SNS but we are checking SQS, is that correct?

Reply:
Just copy from the java tests, will change it to SNS.